aiaccel/hpo/optuna/samplers/nelder_mead_sampler.py

```diff
@@ -268,7 +268,12 @@ def after_trial(
             )
         if isinstance(values, list):
             self.nm.put_value(
-                np.array(list(trial.params.values())),
+                np.array(
+                    [
+                        (value - low) / (high - low)
+                        for value, (low, high) in zip(trial.params.values(), self._search_space.values(), strict=False)
+                    ]
+                ),
```

Comment:
長くなってしまうので、iterator分けましょう。
```suggestion
            it = zip(trial.params.values(), self._search_space.values(), strict=False)
            ...
                np.array([(value - low) / (high - low) for value, (low, high) in it]),
```